packages/@o3r/store-sync/src/sync-storage/interfaces.ts

```diff
@@ -61,4 +61,6 @@ export interface SyncStorageConfig {
   logger?: Logger;
   /** Post process after sync storage execution */
   postProcess?: (state: any) => void;
+  /** List of action types to invoke the reducer */
```

Comment:
This is not really clear, you should specify what is the reducer you are talking about

Reply:
Added more description